phoenix-core/src/main/java/org/apache/phoenix/util/ServerUtil.java

```diff
@@ -225,6 +227,21 @@ public static Table getHTableForCoprocessorScan (RegionCoprocessorEnvironment en
         }
         return getTableFromSingletonPool(env, tableName);
     }
+
+    public static Table getHTableSystemChildLink (RegionCoprocessorEnvironment env) throws IOException {
+        if (coprocessorScanWorks(env)) {
```

Comment:
Looks like coprocessorScanWorks will return true for any HBase version >= 0.98.6, and the earliest version we currently support is 1.3. Can we rid of the getTableFromSingletonPool calls?

Reply:
It seems yes we can, nice catch @gjacoby126.
Thank you for the review, I have updated the PR.